Delfino/DualCore/SmartWinchDualCore/main.c

```diff
@@ -100,6 +100,17 @@ void init_settings()
 
     modbus_holding_regs[Current_Waypoints_Pointer] = 0;
     modbus_holding_regs[Dwell_Time] = 1;
+    
+    //Defines motor coordinates
+    //5 input lengths required in metres as follows:
+    //Motor A to Motor B
+    //Motor C to Motor D
+    //Motor A to Motor D
+    //Motor A to Motor C
+    //Motor B to Motor D
+    //This fully charactises the layout and coordinates are set to allow for...
+    //any quadrilateral motor shape
+    set_motor_coord(len_ab,len_cd,len_ad,len_ac,len_bd);
```

Comment:
you need to access the modbus_regs properly, for example:
replace len_ab with modbus_holding_regs[len_ab]